docs/api-reference/carto/carto-layer.md

```diff
@@ -164,6 +164,59 @@ Optional. Aggregation SQL expression. Only used for spatial index datasets.
 
 Optional. Aggregation resolution level. Only used for spatial index datasets, defaults to 6 for quadbins, 4 for h3.
 
+##### `queryParameters` (Depends on provider)
+
+Optional. Use it to parametrize SQL queries. The format depends on the source's provider, some examples:
+
+
+##### - [PostgreSQL and Redshift](https://node-postgres.com/features/queries):
+```js
+const layer = new CartoLayer({
+     data: `select * from users where username=$1`
+     queryParameters: ['alasarr']
+})
+```
+
+##### - [BigQuery positional](https://cloud.google.com/bigquery/docs/parameterized-queries#node.js):
+```js
+const layer = new CartoLayer({
+     data: `select * from users where username=?`,
+     queryParameters: ['alasarr']
+})
+```
+
+##### - [BigQuery named parameters](https://cloud.google.com/bigquery/docs/parameterized-queries#node.js):
+```js
+const layer = new CartoLayer({
+     data: `select * from users where username=@username`,
+     queryParameters: [{ username: 'alasarr' }]
```

Comment:
Those are only examples, is there any reference what is really supported?
Like, that is strange that "named" parameters are in array.
It would be enough to for them to be just `{ username: 'alasarr' }`

Reply:
yes, actually this is a mistake on my side, it should be just an object where the key is the "name" and the value is the actual value

Reply:
[change made here](https://github.com/visgl/deck.gl/pull/7155/commits/d204292bae70d4bc4795e90cf67cec7c52f40c6d)